discovery/src/main/java/com/alibaba/fescar/discovery/registry/EurekaRegistryServiceImpl.java

```diff
@@ -199,27 +206,25 @@ private static Properties getEurekaProperties(boolean needRegister) {
         return eurekaProperties;
     }
 
-    private static String getApplicationName() {
+    private String getApplicationName() {
         String application = FILE_CONFIG.getConfig(getEurekaApplicationFileKey());
         if (null == application) {
             application = DEFAULT_APPLICATION;
         }
         return application;
     }
 
-    private static EurekaClient getEurekaClient(boolean needRegister) throws EurekaRegistryException{
+    private EurekaClient getEurekaClient(boolean needRegister) throws EurekaRegistryException {
         if (eurekaClient == null) {
             synchronized (EurekaRegistryServiceImpl.class) {
                 try {
                     if (eurekaClient != null) {
```

Comment:
move ，double check singleton

Reply:
done,the non-Null check is not necessary here.